kraken-wrapper/src/kraken/wrapper/_buildenv_uv.py

```
@@ -32,7 +32,7 @@ def _get_create_venv_command(self, python_bin: Path, path: Path) -> list[str]:
 
     def _get_install_command(self, venv_dir: Path, requirements: RequirementSpec, env: dict[str, str]) -> list[str]:
         env["VIRTUAL_ENV"] = str(venv_dir)
-        return [self._uv_bin, "pip", "install", *requirements.to_args(base_dir=self._project_root)]
+        return [self._uv_bin, "pip", "install", "--no-config", *requirements.to_args(base_dir=self._project_root)]
```

Would it be possible to test this?

---

UV only warns when it can't access an index so we would need to build a corner case to properly fail if the option is not set. I am not sure it's worth the time (I got an error locally because of this issue but I am not sure what is actually the cause of it so it's hard for me to create a minimal test to reproduce it).

---

Would something like this help? (from [warnings doc](https://docs.python.org/3/library/warnings.html#warnings.filterwarnings))
```
warnings.filterwarnings("error", category=CorrespondingWarningType)
```

Not saying we should do it now, just for my curiosity.

---

The warning is printed by UV that is in a child process, not sure it will help